.github/workflows/signable-rspec.yml

```
@@ -0,0 +1,34 @@
+name: CI RSpec Tests
+
+on: [push, pull_request]
```

```suggestion
on: 
```
Don't think this is needed 🤔

---

But is it doing the same as line 4-9?
I think this `on` bit specifies "when" we want the action to commence, and the bits on 4-9 specify which paths the things listed in `on` should occur at for the action to commence. 🤔

---

I've never setup a GitHub Action before.
I imitated this example:
https://dev.to/sulmanweb/github-actions-run-rspec-test-on-every-push-for-free-1pb6
But something's wrong because GitHub is giving us an error on the page for that Action:
```
.github/workflows/signable-rspec.yml#L3
You have an error in your yaml syntax on line 3
```

---

I'll try making your suggested change and seeing if that error message goes away...

---

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
I think it is able to get the "when" from the keys on line 4 and 7.

I'm assuming this as we're currently having a syntax error in the Github action.
https://github.com/smartpension/signable/actions/runs/885814467/workflow

---

I think you were right, I've made your change and the error that was there is gone, to be replaced with:
```
.github/workflows/signable-rspec.yml#L1
No steps defined in `steps` for the following jobs: build
```

---

I'll try adding this:
```
    steps:
      uses: actions/build@v3
```

---

More changes in the error messages.
I'll need to take some time to read through those docs.